client/src/components/requests/RequestedList.tsx

```diff
@@ -0,0 +1,169 @@
+                            </Step.Content>
+                        </Step>
+                        <Step disabled>
+                            <Step.Content>
+                                <Step.Title>Approved</Step.Title>
+                            </Step.Content>
+                        </Step>
+                        <Step disabled>
+                            <Step.Content>
+                                <Step.Title>Ready for
+                                    Pickup</Step.Title>
+                            </Step.Content>
+                        </Step>
+                    </Step.Group>);
+            } else if (r.status === 'APPROVED') {
+                steps = (
+                    <Step.Group stackable='tablet' size='mini'>
+                        <Step>
+                            <Step.Content>
+                                <Step.Title>Submitted</Step.Title>
+                            </Step.Content>
+                        </Step>
+                        <Step active>
+                            <Step.Content>
+                                <Step.Title>Approved</Step.Title>
+                            </Step.Content>
+                        </Step>
+                        <Step disabled>
+                            <Step.Content>
+                                <Step.Title>Ready for
+                                    Pickup</Step.Title>
+                            </Step.Content>
+                        </Step>
+                    </Step.Group>);
+            } else if (r.status === 'READY_FOR_PICKUP') {
+                steps = (
+                    <Step.Group stackable='tablet' size='mini'>
+                        <Step>
+                            <Step.Content>
+                                <Step.Title>Submitted</Step.Title>
+                            </Step.Content>
+                        </Step>
+                        <Step>
+                            <Step.Content>
+                                <Step.Title>Approved</Step.Title>
+                            </Step.Content>
+                        </Step>
+                        <Step active>
+                            <Step.Content>
+                                <Step.Title>Ready for
+                                    Pickup</Step.Title>
+                            </Step.Content>
+                        </Step>
+                    </Step.Group>);
+            } else if (r.status === 'FULFILLED' && r.item.returnRequired) {
+                steps = (
+                    <Step.Group stackable='tablet' size='mini'>
+                        <Step active>
+                            <Step.Content>
+                                <Step.Title>Fulfilled</Step.Title>
+                            </Step.Content>
+                        </Step>
+                        <Step disabled>
+                            <Step.Content>
+                                <Step.Title>Returned</Step.Title>
```

Comment:
Can we make it very clear that items require return? (e.g., highlight it in yellow like Photo ID Required is on the hardware desk screen) and maybe show an icon to draw attention to it